### ISSUE TITLE
How can a listener be removed?

### ISSUE THREAD

**Issue by arikhalperin** [matrix-construct/construct#157]
I configured several listeners. It is unclear from documentation how a listener can be removed.

**Comment by jevolk** [comment]
Thanks. Indeed there is no command within the `net listen` interface to do this and one should be added; I'll be keeping this issue open for that. However, you are not out of luck due to the fact that everything in Construct is a matrix event: you can still remove a listener by redacting its event.

For each named listener (for example `net listen matrix` created one listener named `matrix`) there will be a state-event in your `!ircd` room with a type of `ircd.listen` and in the above example a state_key of `matrix`. After redacting that (you may need a server restart) the listener will be removed.

examples:
```
redact !ircd:zemos.net ircd.listen matrix
redact !ircd:zemos.net ircd.listen my_second_listener
```

**Comment by jevolk** [comment]
Implemented by 7434a4b6db; command is `net listen del <name>`
